sources/packages/backend/apps/api/src/auth/roles.enum.ts

```diff
@@ -30,4 +30,5 @@ export enum Role {
   InstitutionCreateNote = "institution-create-note",
   InstitutionApproveDeclineDesignation = "institution-approve-decline-designation",
   InstitutionApproveDeclineOfferingChanges = "institution-approve-decline-offering-changes",
+  RestrictMSFAAReissue = "restrict-msfaa-reissue",
```

Comment:
The roles are created usually prefixed with a prefix (e.g. `AEST`, `Student` or `Institution`) and they state what the user is allowed to do in case it has the role. For this case, trying to follow the other, I believe that it would be `student-reissue-msfaa`. Thoughts?